### ISSUE TITLE
Deployment error while following the Install Tutorial

### ISSUE THREAD

**Issue by StefanoAgazzi** [webiny/webiny-js#1517]
## This is:
- Bug

## Specifications

  - version: latest version available as of 2021-03-17 (how can I get the version number?)
  - OS: macOS Big Sur 11.2.3 (Intel)
  - Browser: none

## Expected Behavior
Successfully complete the deployment as explained in the [Install Tutorial](https://docs.webiny.com/docs/tutorials/install-webiny/)

## Actual Behavior
Deployment fails complaining about an S3 Access Denied issue even if the profile used has `AWS AdministratorAccess` permissions.

```
 +  aws:elasticsearch:Domain webiny-js created
 +  pulumi:pulumi:Stack api-dev creating error: update failed
 +  pulumi:pulumi:Stack api-dev **creating failed** 1 error

Diagnostics:
  aws:s3:BucketObject (./pbInstallation.zip):
    error: 1 error occurred:
    	* Error putting object in S3 bucket (fm-bucket-9f48914): AccessDenied: Access Denied
    	status code: 403, request id: X1NA53PHDJ38YX3H, host id: wVlD/Sce0IUACmePZJFEH99QxFVEFvNqGUJklwm2+uJj3t2i/4EoATxjT3sj0toKwb6+pbNmP3o=

  pulumi:pulumi:Stack (api-dev):
    error: update failed

Resources:
    + 20 created

Duration: 14m21s

webiny error: Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
webiny debug:  Error: Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
    at makeError (/Users/ste/workspaces/isdc/cms/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/Users/ste/workspaces/isdc/cms/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (/Users/ste/workspaces/isdc/cms/node_modules/@webiny/cli-plugin-deploy-pulumi/app/deploy.js:153:9)
    at async Object.handler (/Users/ste/workspaces/isdc/cms/node_modules/@webiny/cli-plugin-deploy-pulumi/app/index.js:46:21) {
  shortMessage: 'Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  command: '/Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  exitCode: 255,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
webiny error: Command failed with exit code 1: yarn webiny deploy api --env dev --debug false --build true --preview false
```

## Steps to Reproduce the Problem

1. **`npx create-webiny-project cms`**
```
Initializing a new Webiny project in /Users/ste/workspaces/isdc/cms...
  ✔ Prepare project folder
  ✔ Setup yarn@^2
  ✔ Install template package
  ✔ Initialize git
Starting @webiny/cwp-template-aws template ...

In order to setup your new Webiny project, please answer the following questions.
```

2. **`? Please choose the AWS region in which your new project will be deployed: eu-west-1 (EU, Ireland)`**
```
⏳ Installing dependencies....

➤ YN0000: ┌ Resolution step
➤ YN0061: │ eslint-plugin-standard@npm:5.0.0 is deprecated: standard 16.0.0 and eslint-config-standard 16.0.0 no longer require the eslint-plugin-standard package. You can remove it from your dependencies with 'npm rm eslint-plugin-standard'. More info here: https://github.com/standard/standard/issues/1316
➤ YN0061: │ i18n-locales@npm:0.0.2 is deprecated: Please upgrade to v0.0.3+ as we have fixed missing ISO-639-1 codes
➤ YN0061: │ eslint-loader@npm:3.0.2 is deprecated: This loader has been deprecated. Please use eslint-webpack-plugin
➤ YN0061: │ urix@npm:0.1.0 is deprecated: Please see https://github.com/lydell/urix#deprecated
➤ YN0013: │ @webiny/ui@npm:5.1.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0061: │ chokidar@npm:2.1.8 is deprecated: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
➤ YN0061: │ deep-assign@npm:3.0.0 is deprecated: Check out `lodash.merge` or `merge-options` instead.
➤ YN0061: │ @babel/polyfill@npm:7.12.1 is deprecated: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
➤ YN0061: │ core-js@npm:2.6.12 is deprecated: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
➤ YN0061: │ resolve-url@npm:0.2.1 is deprecated: https://github.com/lydell/resolve-url#deprecated
➤ YN0013: │ @rmwc/textfield@file:./rmwc/textfield#./rmwc/textfield::hash=de95ae&locator=%40webiny%2Fui%40npm%3A5.1.0 can't be
➤ YN0061: │ react-images@npm:0.5.19 is deprecated: This beta version is no longer supported. Please update to 1.1.0-beta.4
➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
➤ YN0032: │ nan@npm:2.14.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ request@npm:2.88.2 is deprecated: request has been deprecated, see https://github.com/request/request/issues/3142
➤ YN0061: │ html-webpack-plugin@npm:4.0.0-beta.5 is deprecated: please switch to a stable version
➤ YN0061: │ har-validator@npm:5.1.5 is deprecated: this library is no longer supported
➤ YN0061: │ request-promise-native@npm:1.0.9 is deprecated: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
➤ YN0061: │ buffer@npm:4.9.1 is deprecated: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
➤ YN0032: │ evp_bytestokey@npm:1.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0002: │ @apollo/react-hooks@npm:3.1.5 [4eab8] doesn't provide apollo-utilities (pc5536), requested by @apollo/react-common
➤ YN0002: │ @apollo/react-hooks@npm:3.1.5 [5b07d] doesn't provide apollo-utilities (pe4b27), requested by @apollo/react-common
➤ YN0002: │ @apollo/react-hooks@npm:3.1.5 [a9542] doesn't provide apollo-utilities (p6a2eb), requested by @apollo/react-common
➤ YN0002: │ @webiny/api-headless-cms@npm:5.1.0 doesn't provide graphql (p8d1f3), requested by @graphql-tools/schema
➤ YN0002: │ @webiny/api-prerendering-service@npm:5.1.0 doesn't provide puppeteer-core (p1069d), requested by chrome-aws-lambda
➤ YN0060: │ @webiny/app-page-builder@npm:5.1.0 provides react-dnd (ped849) with version 9.5.1, which doesn't satisfy what react-sortable-tree and some of its descendants request
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (p42fcd), requested by apollo-link
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (p459f1), requested by apollo-cache
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (p18a18), requested by @apollo/react-components
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (paf70a), requested by apollo-cache-inmemory
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (p25ca3), requested by apollo-client
➤ YN0002: │ admin@workspace:apps/admin/code doesn't provide graphql (p2eb75), requested by apollo-link-batch-http
➤ YN0002: │ apollo-link-batch@npm:1.1.15 doesn't provide graphql (pc3653), requested by apollo-link
➤ YN0002: │ apollo-link-context@npm:1.0.20 doesn't provide graphql (pa199c), requested by apollo-link
➤ YN0002: │ rc-trigger@npm:2.6.5 doesn't provide react (p8a8f0), requested by rc-animate
➤ YN0002: │ rc-trigger@npm:2.6.5 doesn't provide react-dom (p1433f), requested by rc-animate
➤ YN0002: │ react-dev-utils@npm:10.2.1 doesn't provide typescript (p62ad2), requested by fork-ts-checker-webpack-plugin
➤ YN0002: │ react-dev-utils@npm:10.2.1 doesn't provide webpack (p2e56c), requested by fork-ts-checker-webpack-plugin
➤ YN0002: │ react-scrolllock@npm:2.0.7 doesn't provide react (p6ac0d), requested by react-prop-toggle
➤ YN0002: │ react-scrolllock@npm:2.0.7 doesn't provide react-dom (pc58d4), requested by react-prop-toggle
➤ YN0002: │ theme@workspace:apps/theme doesn't provide @types/react (p774d2), requested by @apollo/react-components
➤ YN0002: │ theme@workspace:apps/theme doesn't provide apollo-cache (p1ef5e), requested by @apollo/react-components
➤ YN0002: │ theme@workspace:apps/theme doesn't provide apollo-client (p32b13), requested by @apollo/react-components
➤ YN0002: │ theme@workspace:apps/theme doesn't provide apollo-link (p210ed), requested by @apollo/react-components
➤ YN0002: │ theme@workspace:apps/theme doesn't provide apollo-utilities (p4b8bc), requested by @apollo/react-components
➤ YN0002: │ theme@workspace:apps/theme doesn't provide prop-types (p2d2f0), requested by react-hamburger-menu
➤ YN0002: │ theme@workspace:apps/theme doesn't provide react-dom (p98606), requested by @apollo/react-components
➤ YN0002: │ website@workspace:apps/website/code doesn't provide apollo-cache (p86d46), requested by @apollo/react-components
➤ YN0002: │ website@workspace:apps/website/code doesn't provide apollo-utilities (p10aef), requested by @apollo/react-components
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 15s 756ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ yocto-queue@npm:0.1.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zen-observable-ts@npm:0.8.21 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zen-observable@npm:0.8.15 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zerop@npm:1.0.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ zip-local@npm:0.3.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed in 4s 792ms
➤ YN0000: ┌ Link step
➤ YN0006: │ cms@workspace:. lists build scripts, but is referenced through a soft link. Soft links don't support build scripts, so they'll be ignored.
➤ YN0007: │ cms@workspace:. must be built because it never did before or the last one failed
➤ YN0007: │ node-sass@npm:4.14.1 must be built because it never did before or the last one failed
➤ YN0007: │ fsevents@patch:fsevents@npm%3A1.2.13#builtin<compat/fsevents>::version=1.2.13&hash=11e9ea must be built because it never did before or the last one failed
➤ YN0007: │ core-js@npm:3.9.1 must be built because it never did before or the last one failed
➤ YN0008: │ @pulumi/aws@npm:3.32.0 must be rebuilt because its dependency tree changed
➤ YN0007: │ core-js@npm:2.6.12 must be built because it never did before or the last one failed
➤ YN0007: │ @fortawesome/fontawesome-common-types@npm:0.2.35 must be built because it never did before or the last one failed
➤ YN0007: │ @fortawesome/fontawesome-svg-core@npm:1.2.34 must be built because it never did before or the last one failed
➤ YN0007: │ @fortawesome/free-brands-svg-icons@npm:5.15.2 must be built because it never did before or the last one failed
➤ YN0007: │ @fortawesome/free-regular-svg-icons@npm:5.15.2 must be built because it never did before or the last one failed
➤ YN0007: │ @fortawesome/free-solid-svg-icons@npm:5.15.2 must be built because it never did before or the last one failed
➤ YN0000: └ Completed in 1m 21s
➤ YN0000: Done with warnings in 1m 42s

🎉 Your new Webiny project cms is ready!

Finish the setup by running the following command: cd cms && yarn webiny deploy

To see all of the available CLI commands, run yarn webiny --help in your cms directory.

Want to delve deeper into Webiny? Check out https://docs.webiny.com!
Like the project? Star us on https://github.com/webiny/webiny-js!

Need help? Join our Slack community! https://www.webiny.com/slack

🚀 Happy coding!
```

3. **`adding AWS_PROFILE=fv-ste in .env`**
```
❯ cd cms
❯ nvim .env
```

4. **`setting node version (not required in the tutorial and probably useless for this)`**
```
❯ nvm use 14
Now using node v14.16.0 (npm v7.6.0)
❯ node -v > .nvmrc
```

5. **`AWS_PROFILE=fv-ste yarn webiny deploy`**
```
It looks like this is your first time using @webiny/pulumi-sdk.
✔ Pulumi downloaded, continuing...
[resource plugin aws-3.32.0] installing
Downloading plugin: 70.36 MiB / 70.36 MiB [=========================] 100.00% 8s

webiny info: This is your first time deploying the project (dev environment). Note that the initial deployment can take up to 15 minutes, so please be patient.

webiny info: Deploying api project application...
webiny info: Running 'build' in 11 packages...
api-file-manager-transform: webiny log: Start bundling
api-dynamo-to-elastic: webiny log: Start bundling
api-file-manager-manage: webiny log: Start bundling
api-graphql: webiny log: Start bundling
api-page-builder-flush: webiny log: Start bundling
api-dynamo-to-elastic: ℹ Compiling DynamoToElastic
api-file-manager-transform: ℹ Compiling Transform
api-page-builder-render: webiny log: Start bundling
api-file-manager-manage: ℹ Compiling Manage
api-page-builder-update-settings: webiny log: Start bundling
api-graphql: ℹ Compiling Graphql
api-file-manager-download: webiny log: Start bundling
api-headless-cms: webiny log: Start bundling
api-page-builder-flush: ℹ Compiling Flush
api-prerendering-service-queue-process: webiny log: Start bundling
api-page-builder-render: ℹ Compiling Render
api-page-builder-update-settings: ℹ Compiling UpdateSettings
api-prerendering-service-queue-add: webiny log: Start bundling
api-file-manager-download: ℹ Compiling Download
api-headless-cms: ℹ Compiling HeadlessCMS
api-prerendering-service-queue-process: ℹ Compiling Process
api-prerendering-service-queue-add: ℹ Compiling Add
api-file-manager-manage: ✔ Manage: Compiled successfully in 4.89s
api-file-manager-manage: webiny log: Finished bundling
api-file-manager-transform: ✔ Transform: Compiled successfully in 4.96s
api-file-manager-transform: webiny log: Finished bundling
api-page-builder-flush: ✔ Flush: Compiled successfully in 4.78s
api-page-builder-flush: webiny log: Finished bundling
api-file-manager-download: ✔ Download: Compiled successfully in 4.55s
api-file-manager-download: webiny log: Finished bundling
api-dynamo-to-elastic: ✔ DynamoToElastic: Compiled successfully in 5.14s
api-dynamo-to-elastic: webiny log: Finished bundling
api-page-builder-update-settings: ✔ UpdateSettings: Compiled successfully in 4.86s
api-page-builder-update-settings: webiny log: Finished bundling
api-prerendering-service-queue-add: ✔ Add: Compiled successfully in 4.50s
api-prerendering-service-queue-add: webiny log: Finished bundling
api-prerendering-service-queue-process: ✔ Process: Compiled successfully in 4.69s
api-prerendering-service-queue-process: webiny log: Finished bundling
api-page-builder-render: ✔ Render: Compiled successfully in 5.05s
api-page-builder-render: webiny log: Finished bundling
api-headless-cms: ✔ HeadlessCMS: Compiled successfully in 4.88s
api-headless-cms: webiny log: Finished bundling
api-graphql: ✔ Graphql: Compiled successfully in 5.43s
api-graphql: webiny log: Finished bundling
webiny info: Running "hook-before-deploy" hook...
webiny info: Using profile fv-ste in eu-west-1 region.
webiny success: Found Elastic Search service role!
webiny success: Hook "hook-before-deploy" completed. Deploying...
Updating (dev):

 +  pulumi:pulumi:Stack api-dev creating
 +  aws:iam:Role default-lambda-role creating
 +  aws:apigatewayv2:Api api-gateway creating
 +  aws:cloudwatch:EventRule ps-process-queue-event-rule creating
 +  aws:iam:Role headless-cms-lambda-role creating
 +  aws:s3:Bucket fm-bucket creating
 +  aws:dynamodb:Table webiny-es creating
 +  aws:dynamodb:Table webiny creating
 +  aws:iam:Role api-lambda-role creating
 +  aws:iam:Role fm-lambda-role creating
 +  aws:elasticsearch:Domain webiny-js creating
 +  aws:cognito:UserPool api-user-pool creating
 +  aws:cloudwatch:EventRule ps-process-queue-event-rule created
 +  aws:apigatewayv2:Api api-gateway created
 +  aws:apigatewayv2:Stage default creating
 +  aws:cognito:UserPool api-user-pool created
 +  aws:cognito:UserPoolClient api-user-pool-client creating
 +  aws:cognito:UserPoolClient api-user-pool-client created
 +  aws:apigatewayv2:Stage default created
 +  aws:cloudfront:Distribution api-cloudfront creating
 +  aws:iam:Role fm-lambda-role created
 +  aws:iam:RolePolicyAttachment fm-lambda-role-policy creating
 +  aws:iam:Role headless-cms-lambda-role created
 +  aws:iam:RolePolicyAttachment headless-cms-lambda-role-policy creating
 +  aws:cloudfront:Distribution api-cloudfront created
 +  aws:iam:RolePolicyAttachment fm-lambda-role-policy created
 +  aws:iam:RolePolicyAttachment headless-cms-lambda-role-policy created
 +  aws:iam:Role api-lambda-role created
 +  aws:iam:RolePolicyAttachment api-lambda-role-policy creating
 +  aws:s3:Bucket fm-bucket created
 +  aws:s3:BucketObject ./pbInstallation.zip creating
 +  aws:lambda:Function fm-manage creating
 +  aws:lambda:Function fm-image-transformer creating
 +  aws:iam:RolePolicyAttachment api-lambda-role-policy created
 +  aws:s3:BucketObject ./pbInstallation.zip creating error: 1 error occurred:
 +  aws:s3:BucketObject ./pbInstallation.zip **creating failed** error: 1 error occurred:
 +  aws:iam:Role default-lambda-role created
 +  aws:dynamodb:Table webiny-es created
 +  aws:lambda:Function fm-manage created
 +  aws:dynamodb:Table webiny created
 +  aws:lambda:Function fm-image-transformer created
 +  aws:elasticsearch:Domain webiny-js created
 +  pulumi:pulumi:Stack api-dev creating error: update failed
 +  pulumi:pulumi:Stack api-dev **creating failed** 1 error

Diagnostics:
  aws:s3:BucketObject (./pbInstallation.zip):
    error: 1 error occurred:
    	* Error putting object in S3 bucket (fm-bucket-9f48914): AccessDenied: Access Denied
    	status code: 403, request id: X1NA53PHDJ38YX3H, host id: wVlD/Sce0IUACmePZJFEH99QxFVEFvNqGUJklwm2+uJj3t2i/4EoATxjT3sj0toKwb6+pbNmP3o=

  pulumi:pulumi:Stack (api-dev):
    error: update failed

Resources:
    + 20 created

Duration: 14m21s

webiny error: Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
webiny debug:  Error: Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
    at makeError (/Users/ste/workspaces/isdc/cms/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/Users/ste/workspaces/isdc/cms/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (/Users/ste/workspaces/isdc/cms/node_modules/@webiny/cli-plugin-deploy-pulumi/app/deploy.js:153:9)
    at async Object.handler (/Users/ste/workspaces/isdc/cms/node_modules/@webiny/cli-plugin-deploy-pulumi/app/index.js:46:21) {
  shortMessage: 'Command failed with exit code 255: /Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  command: '/Users/ste/workspaces/isdc/cms/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  exitCode: 255,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
webiny error: Command failed with exit code 1: yarn webiny deploy api --env dev --debug false --build true --preview false
```

**Comment by MaxSamnani** [comment]
i am also having issue in setting up webiny

biny/my-new-demo). If you have not created a project yet, use `pulumi new` to do so
Karims-MacBook-Pro:my-new-demo karimbadruddin$ yarn webiny deploy
webiny info: Deploying api project application...
webiny info: Running 'build' in 11 packages...
api-dynamo-to-elastic: webiny log: Start bundling
api-headless-cms: webiny log: Start bundling
api-file-manager-download: webiny log: Start bundling
api-file-manager-transform: webiny log: Start bundling
api-page-builder-render: webiny log: Start bundling
api-headless-cms: ℹ Compiling HeadlessCMS
api-dynamo-to-elastic: ℹ Compiling DynamoToElastic
api-file-manager-download: ℹ Compiling Download
api-graphql: webiny log: Start bundling
api-file-manager-transform: ℹ Compiling Transform
api-page-builder-update-settings: webiny log: Start bundling
api-page-builder-render: ℹ Compiling Render
api-prerendering-service-queue-add: webiny log: Start bundling
api-graphql: ℹ Compiling Graphql
api-file-manager-manage: webiny log: Start bundling
api-page-builder-flush: webiny log: Start bundling
api-prerendering-service-queue-process: webiny log: Start bundling
api-page-builder-update-settings: ℹ Compiling UpdateSettings
api-prerendering-service-queue-add: ℹ Compiling Add
api-file-manager-manage: ℹ Compiling Manage
api-page-builder-flush: ℹ Compiling Flush
api-prerendering-service-queue-process: ℹ Compiling Process
api-file-manager-download: ✔ Download: Compiled successfully in 3.28s
api-file-manager-download: webiny log: Finished bundling
api-file-manager-transform: ✔ Transform: Compiled successfully in 3.18s
api-file-manager-transform: webiny log: Finished bundling
api-prerendering-service-queue-add: ✔ Add: Compiled successfully in 2.87s
api-prerendering-service-queue-add: webiny log: Finished bundling
api-dynamo-to-elastic: ✔ DynamoToElastic: Compiled successfully in 4.59s
api-dynamo-to-elastic: webiny log: Finished bundling
api-file-manager-manage: ✔ Manage: Compiled successfully in 2.45s
api-file-manager-manage: webiny log: Finished bundling
api-page-builder-flush: ✔ Flush: Compiled successfully in 2.59s
api-page-builder-flush: webiny log: Finished bundling
api-prerendering-service-queue-process: ✔ Process: Compiled successfully in 2.61s
api-prerendering-service-queue-process: webiny log: Finished bundling
api-page-builder-update-settings: ✔ UpdateSettings: Compiled successfully in 3.44s
api-page-builder-update-settings: webiny log: Finished bundling
api-page-builder-render: ✔ Render: Compiled successfully in 4.54s
api-page-builder-render: webiny log: Finished bundling
api-headless-cms: ✔ HeadlessCMS: Compiled successfully in 8.07s
api-headless-cms: webiny log: Finished bundling
api-graphql: ✔ Graphql: Compiled successfully in 7.65s
api-graphql: webiny log: Finished bundling
webiny info: Running "hook-before-deploy" hook...
webiny info: Using profile default in us-east-1 region.
webiny success: Found Elastic Search service role!
webiny success: Hook "hook-before-deploy" completed. Deploying...
Updating (dev):

    pulumi:pulumi:Stack api-dev running 
    aws:iam:Role api-lambda-role  
    aws:iam:Role default-lambda-role  
 +  aws:s3:Bucket fm-bucket creating 
 +  aws:apigatewayv2:Api api-gateway creating 
 +  aws:elasticsearch:Domain webiny-js creating 
 +  aws:cognito:UserPool api-user-pool creating 
    aws:cloudwatch:EventRule ps-process-queue-event-rule  
 +  aws:dynamodb:Table webiny-es creating 
 +  aws:dynamodb:Table webiny creating 
    aws:iam:Role fm-lambda-role  
    aws:iam:Role headless-cms-lambda-role  
    aws:iam:RolePolicyAttachment api-lambda-role-policy  
    aws:iam:RolePolicyAttachment default-lambda-role-policy  
    aws:iam:RolePolicyAttachment fm-lambda-role-policy  
    aws:iam:RolePolicyAttachment headless-cms-lambda-role-policy  
 +  aws:dynamodb:Table webiny creating error: 1 error occurred:
 +  aws:dynamodb:Table webiny **creating failed** error: 1 error occurred:
 +  aws:cognito:UserPool api-user-pool creating error: 1 error occurred:
 +  aws:cognito:UserPool api-user-pool **creating failed** error: 1 error occurred:
 +  aws:apigatewayv2:Api api-gateway creating error: 1 error occurred:
 +  aws:apigatewayv2:Api api-gateway **creating failed** error: 1 error occurred:
 +  aws:dynamodb:Table webiny-es creating error: 1 error occurred:
 +  aws:dynamodb:Table webiny-es **creating failed** error: 1 error occurred:
 +  aws:s3:Bucket fm-bucket creating error: 1 error occurred:
 +  aws:s3:Bucket fm-bucket **creating failed** error: 1 error occurred:
 +  aws:elasticsearch:Domain webiny-js creating error: 1 error occurred:
 +  aws:elasticsearch:Domain webiny-js **creating failed** error: 1 error occurred:
    pulumi:pulumi:Stack api-dev running error: update failed
    pulumi:pulumi:Stack api-dev **failed** 1 error
 
Diagnostics:
  aws:apigatewayv2:Api (api-gateway):
    error: 1 error occurred:
        * error creating API Gateway v2 API: SubscriptionRequiredException:
        status code: 403, request id: ffb8d5d2-7a15-4969-8eee-d7f4ab2e31dc
 
  aws:dynamodb:Table (webiny-es):
    error: 1 error occurred:
        * error creating DynamoDB Table: SubscriptionRequiredException: The AWS Access Key Id needs a subscription for the service
        status code: 400, request id: 4O6PU7F9NOS33A9PHP94VMNM23VV4KQNSO5AEMVJF66Q9ASUAAJG
 
  aws:elasticsearch:Domain (webiny-js):
    error: 1 error occurred:
        * Error creating ElasticSearch domain: SubscriptionRequiredException:
        status code: 403, request id: b3e5b443-dacb-490e-b7e3-97c051db3db4
 
  pulumi:pulumi:Stack (api-dev):
    error: update failed
 
  aws:cognito:UserPool (api-user-pool):
    error: 1 error occurred:
        * error creating Cognito User Pool: SubscriptionRequiredException: The AWS Access Key Id needs a subscription for the service
        status code: 400, request id: 11306c38-e61c-4d86-bd0a-becbd49453d7
 
  aws:dynamodb:Table (webiny):
    error: 1 error occurred:
        * error creating DynamoDB Table: SubscriptionRequiredException: The AWS Access Key Id needs a subscription for the service
        status code: 400, request id: OVDMDAON8EVG3NR3EGD1BR7LU7VV4KQNSO5AEMVJF66Q9ASUAAJG
 
  aws:s3:Bucket (fm-bucket):
    error: 1 error occurred:
        * Error creating S3 bucket: NotSignedUp: Your account is not signed up for the S3 service. You must sign up before you can use S3.
        status code: 403, request id: DWMN7F0WMQ107S6H, host id: DBSh15yu7d9uj1dUVx0JXlCQPhZueaSUrLRjsJYtciz2c5dK9duQRUuSr1kzloPx9Yg1mU6sUK8=
 
Resources:
    10 unchanged

Duration: 8s

webiny error: Command failed with exit code 255: /Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
webiny debug:  Error: Command failed with exit code 255: /Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive
    at makeError (/Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (/Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/cli-plugin-deploy-pulumi/app/deploy.js:153:9)
    at async Object.handler (/Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/cli-plugin-deploy-pulumi/app/index.js:46:21) {
  shortMessage: 'Command failed with exit code 255: /Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  command: '/Users/karimbadruddin/Downloads/Webiny/my-new-demo/node_modules/@webiny/pulumi-sdk/pulumi/darwin/pulumi/pulumi up --yes --skip-preview --non-interactive',
  exitCode: 255,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
webiny error: Command failed with exit code 1: yarn webiny deploy api --env dev --debug false --build true --preview false

can anyone help?

**Comment by adrians5j** [comment]
@MaxSamnani from the logs, I can see:

> SubscriptionRequiredException: The AWS Access Key Id needs a subscription for the service

https://docs.webiny.com/docs/tutorials/install-webiny/#the-aws-access-key-id-needs-a-subscription-for-the-service

**Comment by MaxSamnani** [comment]
> @MaxSamnani from the logs, I can see:
> 
> > SubscriptionRequiredException: The AWS Access Key Id needs a subscription for the service
> 
> https://docs.webiny.com/docs/tutorials/install-webiny/#the-aws-access-key-id-needs-a-subscription-for-the-service

Hi doitadrian,

can you tell me how many days it takes to get the subscription?

**Comment by adrians5j** [comment]
Not sure @MaxSamnani , but what I can suggest is to check this troubleshooting item, maybe it contains something you didn't do on your end:

https://www.webiny.com/docs/how-to-guides/deployment/deploy-your-project#subscriptionrequiredexception-the-aws-access-key-id-needs-a-subscription-for-the-service

**Comment by StefanoAgazzi** [comment]
FYI @doitadrian I've managed to install webiny without any issue on a new AWS account. Actually also the other account where the deploy/installation first failed was quite new.
If you are interested I can provide further details on those two AWS accounts.

**Comment by adrians5j** [comment]
Cool, thank for sharing the info @StefanoAgazzi.

BTW @MaxSamnani , did you managed to get it working? Did the AWS link that's included in [the section](https://www.webiny.com/docs/how-to-guides/deployment/deploy-your-project#subscriptionrequiredexception-the-aws-access-key-id-needs-a-subscription-for-the-service) I provided in my last response, provide any help?

**Comment by StefanoAgazzi** [comment]
@doitadrian ok, I've found what was the issue, I had turned on the S3 option to **Block Public Access settings for this account**
Switched them off, like in the screenshot, and the deployment worked.
![image](https://user-images.githubusercontent.com/2686945/112751300-8a856e00-8fcd-11eb-801a-c73ea379dba8.png)
so the pbInstallation.zip could not be uploaded.

Totally forgot about it, pretty obvious now, but it might worth a mention in the doc or a check/alert during the deployment process.
